src/OrbitGl/TrackManager.cpp

```diff
@@ -142,17 +142,17 @@ void TrackManager::SortTracks() {
 
     last_thread_reorder_.Restart();
 
-    UpdateVisibleTrackList();
+    UpdateFilteredTrackList();
   }
   sorting_invalidated_ = false;
 }
 
 void TrackManager::SetFilter(const std::string& filter) {
   filter_ = absl::AsciiStrToLower(filter);
-  UpdateVisibleTrackList();
+  UpdateFilteredTrackList();
 }
 
-void TrackManager::UpdateVisibleTrackList() {
+void TrackManager::UpdateFilteredTrackList() {
```

Comment:
Why the name change if the function still updates the "visible_tracks_" field? It seems "UpdateVisibleTrackList" better reflects what the function is doing no?

Reply:
It's a plane revert. I didn't make anything else. I will make this PR again taking care of the regressions.